lib/appsignal/demo.ex

```diff
@@ -15,7 +15,7 @@ defmodule Appsignal.Demo do
     |> Appsignal.Transaction.set_error(
       "TestError",
       "Hello world! This is an error used for demonstration purposes.",
-      System.stacktrace
+      []
```

Comment:
this makes the demo error page quite plain. Could we raise an error in this function and catch it to set the stacktrace?

Reply:
Absolutely! https://github.com/appsignal/appsignal-elixir/pull/386/commits/98bbaf03245b5d7d7e3dde9f654dd981f2a0adf9